NEWS.md

```diff
@@ -1,4 +1,14 @@
-## 1.1.1 (UnReleased)
+## 1.1.2 (UnReleased)
+
+## 1.1.1 (Released)
```

Comment:
Please, be careful with versioning. You can use semantic versioning guide https://semver.org/spec/v2.0.0.html. So here I don't see reasons for patch number increasing. It seems minor version should be increased, i.e. 1.2.0.